parlai/core/torch_agent.py

```diff
@@ -720,7 +720,7 @@ def __init__(self, opt: Opt, shared=None):
         self.rank_candidates = opt['rank_candidates']
         self.add_person_tokens = opt.get('person_tokens', False)
         # set interactive mode or not according to options.
-        self.set_interactive_mode(opt['interactive_mode'], shared)
+        self.set_interactive_mode(opt.get('interactive_mode', False), shared)
```

Comment:
when is interactive mode not set?

Reply:
hmm, i ran into this, but can't remember where now, srry